extensions/scarb-doc/src/docs_generation.rs

```diff
@@ -153,3 +153,35 @@ impl_doc_item!(TraitType);
 impl_doc_item!(TraitFunction);
 impl_doc_item!(TypeAlias);
 impl_doc_item!(Variant);
+
+trait ItemsName<'a> {
+    const ITEMS_NAME: &'a str;
+}
```

Comment:
The purpose of this trait is to avoid hardcoding the names multiple times (just hardcode them here)